README.md

```diff
@@ -194,7 +202,7 @@ Download a file.
 ```csharp
 var downloadedFilePath = await Rest.DownloadFileAsync("www.your.api/your_file.pdf");
 
-if (!string.IsNullOrWhiteSpace(downloadedFilePath))
+if (!string.IsNullOrWhiteSpace(downloadedFilePath) && File.Exists(downloadedFilePath))
```

Comment:
Should we log else case?

Reply:
A bit excessive for examples, but added them anyway.

Reply:
I meant for this one example not all 😓